conduct.md

```diff
@@ -24,15 +21,14 @@ As a member of the community, you agree to the following:
 
 **Encouraged:**
 
-- Be kind and courteous.
-- Respect differences of opinion and remember that every design or implementation choice carries a trade-off and numerous costs. There is seldom a single right answer.
-- Remember that everyone was new to Scala at some point. We want to encourage newcomers to join our community and learn the Scala language and ecosystem. Assume competence.
-- Show empathy towards other community members.
+- **Be kind and courteous.** Keep in mind that all public communication is received by many people you don’t know, so before sending a message please ask yourself whether someone from a different context would misunderstand it. Do not attack other persons, in particular consider that age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, or religion have no bearing on the respect you owe another person.
```

Comment:
Was "sexual identity and orientation" dropped from this list on purpose?  I believe those to be distinct from anything else in the list, so if they were dropped because they're subsumed, I think that's a mistake.

Reply:
That would extend the list to:
- gender expression
- gender identity
- sexual identity
- sexual orientation
- sex characteristics

Effective and impactful communication is achieved by a few, carefully chosen words, not by many. May it sound counter-intuitive to some: If you want to communicate all of the points more strongly, I recommend to reduce the same to from five to two, namely "sexuality and gender". This, firstly, reads more welcoming than the alternative, secondly, communicates the points concisely and does not take away from the message, and thirdly, gives you equal grounds to enforce the same.

Reply:
I agree with the gist here, but if you want to capture the full list, you need one more: sexuality, gender and sex characteristics.

Reply:
Could you explain why you think it reads more welcoming?

Reply:
Sure. The language being used communicates not just objective facts, but also mind-set and atmosphere. Firstly, the atmosphere of the paragraph in question has already been well described by @Ichoran:

> it does sound awfully hostile and/or scary in the first few lines. "You're gonna discriminate against someone on the basis of personal appearance, aren't you? Aren't you! [...]"

I think this could in part be remedied by being less verbose in detailing inacceptible behaviour - while retaining the original meaning.

Secondly, to the mind-set. As I'm sure you are well aware, questions surrounding views on gender identity are highly controversial and political. For example, the controversy includes questions about the relationship between gender and sex, and is influenced by factors such as religion and culture. By making a bullet point list that distinguishes in detail between "gender expression", "gender identity", "sexual identity", "sexual orientation" and "sex characteristics" the document is making an implicit, yet unambiguous statement which side of the debate the reader is to hold, and which view is accepted in the community.

The Code of Conduct should prohibit discrimination on controversial grounds, and preclude this community from being a platform for such political debates, but it's language should not be prescriptive of, or siding with, political views, either implicitly, or explicitly.

Reply:
@dwijnand My removal was based on “seeing a duplicate”, an oversight. But I’m glad that I did it, because this discussion has shown further room for improvement, as @otto-dev aptly put it. Let’s make the CoC as precise and at the same time as little political as possible by shortening the list to what @martijnhoekstra suggests.

Reply:
I think changing this list again is a distinct change that would need its own PR with its own discussion. I ask that we leave it out of this PR.

Note that we only just quite recently went around about this at https://github.com/scala/scala-lang/pull/1079#discussion_r326688217 . (The stopping point of that discussion was to use wording taken verbatim from the Contributor Covenant, a respected and widely adopted upstream source, rather than try to wordsmith our own.)

Reply:
We can achieve the reorganization and make the document feel friendlier and more welcoming without changing the contents of this list, so I agree with Seth's point here.

I also think that otto has a good point about the implicit message you're sending by what exactly you put on a list.  But a separate PR would be a better place to discuss what the implications are both in mood and revealed political preference (of the original list, not the modification), and whether they're acceptable; and if not, how we should change them.

That we already talked about whether to adopt the Contributor Covenant language verbatim doesn't mean we shouldn't do so again, but it's a good idea to read the reasoning there so we don't need repeat it on a new PR (or here).

Reply:
Good point, I’ll move the original list verbatim, then we can discuss in a separate PR.